plugins/simpleadminhacks-paper/src/main/java/com/programmerdan/minecraft/simpleadminhacks/framework/utilities/BuildLimit.java

```diff
@@ -0,0 +1,81 @@
+package com.programmerdan.minecraft.simpleadminhacks.framework.utilities;
+
+import org.bukkit.configuration.serialization.ConfigurationSerializable;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class BuildLimit implements ConfigurationSerializable {
+	private String world;
+	private String type;
```

Comment:
`type` should ideally be an enum instead of a string.

Reply:
There doesn't seem to be a `type` other than altitude. Is this necessary?

Reply:
SoundTech asked to add `type` because he wants to add others (e.g radius limit) in the future

Reply:
Making it an enum would mean adding yet another useless for anything else class. Can't make it a subclass because it needs to be accessed from different packages.

Reply:
I do not understand the concern with adding another class, even if it was necessary which I don't think it is? Would defining a public enum in an already existing class not work just fine?

In any case, using a String is quite bad form as:
1. It can be anything - easy for bugs to sneak in
2. Finding out what it can be is near impossible
Enums exist for the exact purpose of fixing these issues.